infra/aws/terraform/modules/eks-prow-iam/boundary.tf

```diff
@@ -35,7 +35,8 @@ data "aws_iam_policy_document" "provisioner_permission_boundary_doc" {
       "kms:*",
       "logs:*",
       "sts:*",
-      "s3:*"
+      "s3:*",
+      "elasticloadbalancing:*"
```

Comment:
I think this is required, otherwise getting following status when trying to create LB svc
```
  Warning  SyncLoadBalancerFailed  11m   service-controller  Error syncing loadbalancer: failed to ensure loadbalancer: AccessDenied: User:arn:aws:sts::468814281478:assumed-role/prow-build-cluster-cluster-20230302162804874400000001/1684763901041941468 is not authorized to perform: elasticloadbalancing:DescribeLoadBalancers because no permissions boundary allows the elasticloadbalancing:DescribeLoadBalancers action status code: 403, request id: ef56a283-6844-4d5f-9112-e20758b9ca8e
  Warning  SyncLoadBalancerFailed  6m4s  service-controller  Error syncing load balancer: failed to ensure load balancer: AccessDenied: User: arn:aws:sts::468814281478:assumed-role/prow-build-cluster-cluster-20230302162804874400000001/1684763901041941468 is not authorized to perform: elasticloadbalancing:CreateLoadBalancer on resource: arn:aws:elasticloadbalancing:us-east-2:468814281478:loadbalancer/ae9fea3f119bb4fe6b15f4ebe8c19fa9 because no permissions boundary allows the elasticloadbalancing:CreateLoadBalancer action
```

Reply:
incorporated as part of https://github.com/kubernetes/k8s.io/pull/5267

Reply:
Correct. In https://github.com/kubernetes/k8s.io/pull/5267 `ProvisionerPermissionBoundary` has been replaced with `EKSResourcesPermissionBoundary`.